Kadane's Algorithm for Maximum Subarray Sum

```diff
@@ -0,0 +1,27 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+  int n;
+  int cs=0,ms=0;//Stores current sum and maximum sum
+  cin>>n;
+  vector<int> a;
+  for(int i=0;i<n;++i)
+  {
+      int x;
+      cin>>x;
+      a.push_back(x);
+  }
+  for(int i=0;i<n;++i)
+  {
+     cs=cs+a[i];
```

Comment:
can we have some good names for `cs` and `ms` so that it can be easily understandable